pktoken/pktoken.go

```diff
@@ -203,19 +223,36 @@ func (p *PKToken) Hash() (string, error) {
 }
 
 func (p *PKToken) MarshalJSON() ([]byte, error) {
-	message := jws.NewMessage().
-		SetPayload(p.Payload).
-		AppendSignature(p.Op).
-		AppendSignature(p.Cic)
-
-	if p.Cos != nil {
-		message.AppendSignature(p.Cos)
+	rawJws := simplejws.Jws{
```

Comment:
if we're creating out own types here that are safe from the issues with the jws library we were using, then why do we need to also save the raw tokens as values when we can replace the jws.Signature with our own and always be able to create them from the object as saved?

Reply:
Eventually I hope to remove all dependencies on `jwx`. That is too large of a change to make in a single PR and impacts too much. This PR introduces simplejws as the acorn to fix the bug and set that feature work up.
The general sketch of this work is:
1. We move to compact tokens internally in the pktoken (this PR)
2. Then we remove the Compact function from pktoken
3. Hopefully by this point we have moved CIC creation and verification into the providers.
4. Slow start moving away from jwx